cfme/web_ui/__init__.py

```diff
@@ -1239,13 +1241,22 @@ def _fill_form_dict(form, values, **kwargs):
 
 
 class Input(Pretty):
-    pretty_attrs = ['name']
+    """Class designed to handle things about ``<input>`` tags that have name attr in one place.
 
-    def __init__(self, name):
-        self.name = name
+    Also applies on ``textarea``, which is basically input with multiple lines (if it has name).
+
+    Args:
+        names: Possible values (or) of the ``name`` attribute.
+    """
+    pretty_attrs = ['names']
+
+    def __init__(self, *names):
+        self.names = names
 
     def locate(self):
-        return '//input[@name="{}"]'.format(self.name)
+        return '(//input | //textarea)[{}]'.format(
+            " or ".join("@name={}".format(quoteattr(name)) for name in self.names)
+        )
```

Comment:
@seandst This was required due to search boxes in host and vms having slightly different name (search_text vs search[text])